### PR TITLE
Sample maintenance

### PR DESCRIPTION
**Pull request by keyboardsurfer**

### CODE REVIEW

File: MacrobenchmarkSample/baseBenchmarks/src/main/java/com/example/benchmark/macro/base/scroll/ScrollBenchmark.kt [hunk 1]
```diff
@@ -38,6 +39,8 @@ import org.junit.runner.RunWith
  * Scrolls down and up in the default activity.
  */
 @RunWith(AndroidJUnit4::class)
+// These tests are only useful for apps that have a scrollable view in the benchmarked activity.
+@Suppress
```
Comment (mlykotom) [inline MacrobenchmarkSample/baseBenchmarks/src/main/java/com/example/benchmark/macro/base/scroll/ScrollBenchmark.kt:43]: Could we leverage JUnit `assumeTrue` for that?

Like:
- getting the UiDevice within the test (but before the `rule.measureRepeated`)
- checking whether `assumeTrue(device.hasObject(By.scrollable()))` (pseudocode)

File: MacrobenchmarkSample/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 #Fri Mar 11 17:00:50 GMT 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
```
Comment (mlykotom) [inline MacrobenchmarkSample/gradle/wrapper/gradle-wrapper.properties:19]:
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
```

**Review by mlykotom — APPROVED** [review]
It would be cool if we could leverage the `assumeTrue` API for skipping the scrollable benchmark. Otherwise LGTM

File: MacrobenchmarkSample/baseBenchmarks/src/main/java/com/example/benchmark/macro/base/scroll/ScrollBenchmark.kt (hunk 1 above)
Comment (keyboardsurfer) [reply MacrobenchmarkSample/baseBenchmarks/src/main/java/com/example/benchmark/macro/base/scroll/ScrollBenchmark.kt:43]: We could find another way but that's currently not a priority. Filed #207 to keep track of this.

**Pull request merged**